Tests/DatadogTests/Datadog/RUM/RUMVitals/VitalCPUReaderTest.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+import UIKit.UIApplication
+@testable import Datadog
+
+class VitalCPUReaderTest: XCTestCase {
+    let testNotificationCenter = NotificationCenter()
+    lazy var cpuReader = VitalCPUReader(notificationCenter: testNotificationCenter)
+
+    func test_whenCPUUnderHeavyLoad_itMeasuresHigherCPUTicks() throws {
+        let highLoadAverage = try averageCPUTicks {
+            for _ in 0...500_000 {
+                let random = Double.random(in: Double.leastNonzeroMagnitude...Double.greatestFiniteMagnitude)
+                _ = tan(random).squareRoot()
+            }
+        }
+
+        let lowLoadAverage = try averageCPUTicks {
+            Thread.sleep(forTimeInterval: 1.0)
+        }
+
+        XCTAssertGreaterThan(highLoadAverage, lowLoadAverage)
+    }
+
+    func test_whenInactiveAppState_itIggnoresCPUTicks() throws {
```

Comment:
typo & naming convention:
```suggestion
    func testWhenInactiveAppStateItIgnoresCPUTicks() throws {
```